openapi/components/schemas/CheckoutForm/CommonCheckoutForm.yaml

```
@@ -129,6 +129,14 @@ properties:
             description: The Heap.io tracking identifier string.
             type: string
             example: 1234567890
+      requiredOptionalFields:
```

The name sounds mindblowing :/

---

Yes, I don't like this name. It's an oxymoron (but not a common figure of speech).

You could change `Optional` to `Additional`.
```suggestion
      requiredAdditionalFields:
```